data.sql

```diff
@@ -3,4 +3,33 @@
 INSERT INTO animals VALUES (1, 'Agumon', '2020-02-03', 0, true, 10.23);
 INSERT INTO animals VALUES (2, 'Gabumon', '2018-11-15', 2, true, 8.00);
 INSERT INTO animals VALUES (3, 'Pikachu', '2021-01-07', 1, false, 15.04);
-INSERT INTO animals VALUES (4, 'Devimon', '2017-05-012', 5, true, 11.00);
\ No newline at end of file
+INSERT INTO animals VALUES (4, 'Devimon', '2017-05-012', 5, true, 11.00);
+
+-- ################################# DAY 2 ######################################
+
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES (5, 'Charmander', '2020-02-08', 0, false, -11), (6, 'Plantmon', '2021-11-15', 2, true, -5.7), (7, 'Squirtle', '1993-04-02', 3, false, -12.13), (8, 'Angemon', '2005-06-12', 1, true, -45), (9, 'Boarmon', '2005-06-07', 7, true, 20.4), (10, 'Blossom', '1998-10-13', 3, true, 17), (11, 'Ditto', '2022-05-14', 4, true, 22);
+
+BEGIN;
+UPDATE animals SET species = 'unspecified'
+ROLLBACK;
+SELECT * from animals;
+
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species IS null;
+COMMIT;
+SELECT * from animals;
+
+BEGIN;
+DELETE FROM animals;
+ROLLBACK;
+SELECT * FROM animals;
+
+vet_clinic=# BEGIN;
+DELETE FROM animals WHERE date_of_birth > '2022-01-01';
+SAVEPOINT deleteYoung;
+UPDATE animals SET weight_kg = weight_kg * -1;
+ROLLBACK TO deleteYoung;
+UPDATE animals SET weight_kg = weight_kg * -1 WHERE weight_kg < 0;
+COMMIT;
+SELECT * FROM animals;
```

Comment:
- [ ] Kindly be noticed that the transaction commands should be in the `queries.sql` file.

Reply:
Noted thank you!